src/include/migraphx/shape.hpp

```diff
@@ -52,6 +53,7 @@ struct MIGRAPHX_EXPORT shape
 #define MIGRAPHX_SHAPE_VISIT_TYPES(m) \
     m(bool_type, bool) \
     m(half_type, half) \
+    m(bf16_type, bf16) \
```

Comment:
It looks like this changes the order of the visit types. @TedThemistokleous do you know if this breaks ABI?

Reply:
Can probably move the new type to the bottom of the list to fix if it does break abi

Reply:
I think the change should happen in migraphx.h

Reply:
Does it break ABI or not?